tests/benchmarks/test_work_stealing.py

```diff
@@ -27,16 +27,18 @@ def test_work_stealing_on_scaling_up(
     upload_cluster_dump,
     benchmark_all,
     dask_env_variables,
+    gitlab_cluster_tags,
 ):
     with Cluster(
         name=test_name_uuid,
         n_workers=1,
         worker_vm_types=["t3.medium"],
```

Comment:
In this case, don't we want to change the worker's vm type too for its m6 equivalent?